### ISSUE TITLE
Remove the sprintf javascript library from ValidForm.js

### ISSUE THREAD

**Issue by rvanbaalen** [validformbuilder/validformbuilder#62]
Validform.js currently uses the `sprintf` javascript library to do 4 string replacements. Each of these replacements can easily be done with a native `"string".replace("%s", value)`.

This is only used to inject the `minLength` and `maxLength` error values in the error message. Seems like huge overkill to use an entire javascript library / utility for that purpose.

What do you think, @flangfeldt ?

**Comment by flangfeldt** [comment]
True, overkill. This was born from the idea that it would be best to mimic the PHP logic and methods as close as possible. Which seems stupid at this point.

**Comment by rvanbaalen** [comment]
@flangfeldt Do you agree we can simply replace this

```js
sprintf(this.minLengthError, this.minLength)
```

with this

```
this.minLengthError.replace('%s', this.minLength)
```
